soc/xtensa/intel_adsp/ace/include/adsp_timestamp.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_INTEL_ADSP_ACE_TIMESTAMP_H_
+#define ZEPHYR_SOC_INTEL_ADSP_ACE_TIMESTAMP_H_
+
+#include <stdint.h>
+
+struct intel_adsp_timestamp {
```

Comment:
Any explanation of what this struct is would be good. I have no idea what those variables are.

Reply:
The fields of `struct intel_adsp_timestamp` stand for the content of DfTTS snapshot registers (DfISCS, DfLSCS, etc). Added the appropriate comment.